src/pages/my/todo.tsx

```diff
@@ -45,12 +44,12 @@ export const getServerSideProps : GetServerSideProps = scaffold(async (context)
 }, scaffoldOptions);
 
 const MyTodoPage : PageWithLayout = () => {
-    const bookedEventsQuery = useQuery('bookedEvents', getBookedEvents());
     const callAssignmentsQuery = useQuery('callAssignments', getCallAssignments());
+    const respondEventsQuery = useQuery('respondEvents', getRespondEvents());
```

Comment:
The name here, `respondEvents` still makes very little sense to me. Now that things are starting to get cleaner, I think it makes sense for this to get a name to reflect what it actually contains, which is (unless I'm mistaken) all the events that the user has either signed up for or been booked on.

How is it different from `getUserEvents()`?

Reply:
Yes, `respondEvents` basically filters out any non-booked or -responsed events, in contrast to `getUserEvents` which gets all user events. I was thinking this filtering could take place in the `MyTodoPage` component based on the data from `getUserEvents`. Is this a good idea?

Reply:
I think the problem here is naming. When you say "user events", to me that signals that it's events that are directly related to the user somehow. But if I understand correctly, it's _all events in all organizations which the user follows_.

So if the user follows 10 organizations, each with 100 events, that 1000 "user events", which sounds weird to me. Maybe something like `getEventsFromFollowedOrganizations` or something like that.

On the other hand, "respond events" are events that the user _either signed up to or have been booked on_, i.e. events that **are** directly related to the user. I would probably call those "user events".

Reply:
Exactly, 

> When you say "user events", to me that signals that it's events that are directly related to the user somehow. But if I understand correctly, it's _all events in all organizations which the user follows_.

Exactly!

Reply:
Very good name changing suggestions! And I understand what you mean regarding the proximity between the user and events in regards to the two functions.